src/systems/bit-media-frames.js

```diff
@@ -133,89 +157,152 @@ const snapToFrame = (() => {
   };
 })();
 
-function setMatrixScale(obj, scaleArray) {
+const setMatrixScale = (() => {
   const position = new THREE.Vector3();
   const quaternion = new THREE.Quaternion();
   const scale = new THREE.Vector3();
   const m4 = new THREE.Matrix4();
-  obj.updateMatrices();
-  obj.matrixWorld.decompose(position, quaternion, scale);
-  setMatrixWorld(obj, m4.compose(position, quaternion, scale.fromArray(scaleArray)));
-}
 
-function cloneForPreview(world, eid) {
-  // TODO We assume capturable object is an AFRAME entity
-  const el = world.eid2obj.get(eid).el;
-  const mesh = el.getObject3D("mesh");
-  const meshClone = cloneObject3D(mesh, false);
-  meshClone.traverse(node => {
-    updateMaterials(node, function (srcMat) {
-      const mat = srcMat.clone();
-      mat.transparent = true;
-      mat.opacity = 0.5;
-      mat.format = THREE.RGBAFormat;
-      mat.blending = THREE.NormalBlending;
-      node.material = mat;
-      return mat;
+  return (obj, scaleArray) => {
+    obj.updateMatrices();
+    obj.matrixWorld.decompose(position, quaternion, scale);
+    setMatrixWorld(obj, m4.compose(position, quaternion, scale.fromArray(scaleArray)));
+  };
+})();
+
+const frame2mixer = new Map();
```

Comment:
To confirm, this `frame2mixer` is used only for `AEntity` (in old entity system) isn't it? I was not sure if it is good to manage `AnimationMixer` in this system, but it may be ok if it is only for old system that we will remove sooner or later. Adding a comment about it may help.

Reply:
Removed as it just adds unnecessary complexity.